client/components/jetpack/upsell-product-wpcom-plan-card/index.tsx

```diff
@@ -0,0 +1,200 @@
+	siteId: number | null;
+	onCtaButtonClick: () => void;
+}
+
+const UpsellProductWpcomPlanCard: React.FC< UpsellProductWpcomPlanCardProps > = ( {
+	nonManageProductSlug,
+	WPcomPlanSlug,
+	siteId,
+	onCtaButtonClick,
+} ) => {
+	const translate = useTranslate();
+	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const item = slugToSelectorProduct( nonManageProductSlug ) as SelectorProduct;
+	const plan = getPlan( WPcomPlanSlug ) as Plan;
+	const planSlug = plan.getStoreSlug();
+	const jetpackFeatures = plan?.get2023PricingGridSignupJetpackFeatures?.();
+	const jetpackFeaturesObject = getPlanFeaturesObject( jetpackFeatures );
+	const siteProduct = useSelector( ( state ) =>
+		getSiteAvailableProduct( state, siteId, planSlug )
+	);
+
+	const billingTerm: Duration = TERM_ANNUALLY;
+	const ctaButtonURL: string = `https://wordpress.com/checkout/${ selectedSiteSlug }/${ planSlug }?redirect_to=${ encodeURIComponent(
+		window.location.href
+	) }`;
+	const isFetchingPrices: boolean = ! siteProduct;
+	const originalPrice: number = siteProduct?.cost_smallest_unit ?? 0;
+	const displayPrice: number = siteProduct?.cost / 12 ?? 0;
+
+	const ctaButtonLabel = translate( 'Get %(productName)s plan', {
+		args: {
+			productName: plan.getTitle(),
+			context: 'The Plan name, ie- Starter, Explorer, Creator,.',
+		},
+	} );
+
+	const upsellImageAlt = translate( 'Get %(productName)s plan', {
+		args: {
+			productName: plan.getTitle(),
+			context: 'The Plan name, ie- Starter, Explorer, Creator,.',
+		},
+		textOnly: true,
+	} );
+
+	const upsellImageUrl = useMemo( () => {
+		if ( isJetpackBackupSlug( nonManageProductSlug ) ) {
+			return BackupImage;
+		}
+		if ( isJetpackScanSlug( nonManageProductSlug ) ) {
+			return ScanImage;
+		}
+		if ( isJetpackSearchSlug( nonManageProductSlug ) ) {
+			return SearchImage;
+		}
+		return DefaultImage;
+	}, [ nonManageProductSlug ] );
+
+	const { displayName, description } = item;
+
+	const renderPriceDetails = () => {
+		return (
+			<>
+				<div className="display-price__details" aria-hidden="true">
+					<span className="display-price__billing-time-frame">
+						{ translate( 'per month, %(rawPrice)s billed annually, excl. taxes', {
+							args: {
+								rawPrice: ! isFetchingPrices
+									? formatCurrency( originalPrice ?? 0, currencyCode ?? '', {
+											stripZeros: true,
+											isSmallestUnit: true,
+									  } )
+									: '',
+							},
+							comment: 'Excl. Taxes is short for excluding taxes',
+						} ) }
+					</span>
+				</div>
+			</>
+		);
+	};
+
+	const renderProductCardBody = () => {
+		return (
+			<>
+				<div className="upsell-product-card__price-plan">Creator plan</div>
+				<div className="upsell-product-card__price-container">
+					<DisplayPrice
+						isFree={ ! isFetchingPrices && originalPrice === 0 }
+						currencyCode={ currencyCode }
+						originalPrice={ displayPrice ?? 0 }
```

Comment:
Note to self, it seems like this gets formatted here: https://github.com/Automattic/wp-calypso/blob/44155e5cdd1e39dc3403c35b5efcf9275b427b70/client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx#L116-L118